README.md

```diff
@@ -0,0 +1,25 @@
+# ethereum-cache-creator
+## How to use program in the current iteration
+* download the [artifact](https://github.com/internet-sicherheit/ethereum-cache-creator/actions/runs/104782193)
+* unpack zip with a extractor of your choice, I would recommend [7zip](https://www.7-zip.de/).
+* open folder/directory in terminal
+* To run the program: 
+```
+java -jar BloxbergCacheCreator.jar <arguments>
```

Comment:
Here should come an additional example about calling it with some actual arguments, like:
`java -jar BloxbergCacheCreator.jar --ui=cli --stop=3000`